htlcswitch/link.go

```diff
@@ -1077,8 +1077,8 @@ func (l *channelLink) htlcManager() {
 
 			l.fail(
 				LinkFailureError{
-					code:       ErrRecoveryError,
-					ForceClose: false,
+					code:          ErrRecoveryError,
+					FailureAction: LinkFailureForceClose,
```

Comment:
should be `LinkFailureForceNone`

Reply:
yes this can be hit if `SendMessage` fails inside of `syncChanStates`

Reply:
Ah yeah, can just not define it.